tests/foreman/cli/test_ansible.py

```diff
@@ -334,6 +335,35 @@ def test_positive_run_concurrent_jobs(self, rex_contenthosts, target_sat):
         target_sat.cli.GlobalParameter().delete({'name': param_name})
         assert len(target_sat.cli.GlobalParameter().list({'search': param_name})) == 0
 
+    @pytest.mark.parametrize('value', [0, -2, 2.5, 'a', '2.5'])
+    @pytest.mark.rhel_ver_list([8])
```

Comment:
Should we test it for all supported rhel versions?

Reply:
The check happens (and should happen) on Satellite server side. For total completeness, it would perhaps be better but I don't think it's worth it.